db/migrate/20210413020416_create_transactions.rb

```diff
@@ -2,7 +2,7 @@ class CreateTransactions < ActiveRecord::Migration[5.2]
   def change
     create_table :transactions do |t|
       t.integer :invoice_id
-      t.integer :credit_card_number
+      t.string :credit_card_number
```

Comment:
Agree that using a string here is best. Read some articles that you should default to a string for any numbers that you don't need to do calculations with, but will be interesting to see what we learn here.